akka-cluster-sharding-typed/src/main/scala/akka/cluster/sharding/typed/ClusterShardingSettings.scala

```diff
@@ -353,7 +356,8 @@ final class ClusterShardingSettings(
       shardRegionQueryTimeout: FiniteDuration,
       stateStoreMode: ClusterShardingSettings.StateStoreMode,
       tuningParameters: ClusterShardingSettings.TuningParameters,
-      coordinatorSingletonSettings: ClusterSingletonManagerSettings) =
+      coordinatorSingletonSettings: ClusterSingletonManagerSettings,
+      leaseSettings: Option[LeaseUsageSettings]) =
```

Comment:
I think we will have to add another constructor rather, to keep source/binary compatiblity, and deprecate it (that will cover the current constructor, which user code may be calling)

Reply:
I have compiled it, but the formatter changed 66 other files.
Should I also add these files as well?
![format](https://user-images.githubusercontent.com/53307459/94434236-bef0ac80-0199-11eb-8eca-fa90abc9e038.png)

Reply:
Hmmm, that is surprising, it should only change the files you have touched (and have failed the PR validation if anyone tried to commit incorrectly formatted sources just like how it did with this PR).

Skip those and only push the files you have changed, after they were reformatted.